native/Makefile

```
@@ -88,6 +88,39 @@ native-all: \
     mac-x86 mac-x86_64 mac-arm64 \
     freebsd-x86 freebsd-x86_64
 
+# Sign macOS native libraries with Apple Developer ID.
+# Requires a "Developer ID Application" certificate in the keychain.
+# Usage: make sign APPLE_SIGNING_IDENTITY="Developer ID Application: Guillaume Nodet (CBCRPRKJ9H)"
+APPLE_SIGNING_IDENTITY ?= Developer ID Application: Guillaume Nodet (CBCRPRKJ9H)
+NATIVE_MAC_DIR := src/main/resources/org/jline/nativ/Mac
+
+sign:
```

checkmake (0.3.2)

[warning] Target body for "sign" exceeds allowed length of 5 lines (10).

(maxbodylength)